### PR TITLE
i18n(fr) Add french transltion for Backend file

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)

Add french translation for `appwriteio.mdx` & `supabase.mdx` & `tigris.mdx`

### CODE REVIEW

File: src/content/docs/fr/guides/backend/supabase.mdx [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+  <p>Already have an account? <a href="/signin">Sign in</a></p>
+  <form action="/api/auth/register" method="post">
+    <label for="email" for="email">Email</label>
+    <input type="email" name="email" id="email" />
+    <label for="password">Password</label>
+    <input type="password" name="password" id="password" />
+    <button type="submit">Login</button>
+  </form>
+</Layout>
+```
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/backend/supabase.mdx:265]:
```suggestion
<Layout title="S'enregistrer">
  <h1>S'enregistrer</h1>
  <p>Vous avez déjà un compte ? <a href="/signin">Connexion</a></p>
  <form action="/api/auth/register" method="post">
    <label for="email" for="email">Email</label>
    <input type="email" name="email" id="email" />
    <label for="password">Mot de passe</label>
    <input type="password" name="password" id="password" />
    <button type="submit">Se connecter</button>
  </form>
</Layout>
```
```

File: src/content/docs/fr/guides/backend/supabase.mdx [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+  <p>New here? <a href="/register">Create an account</a></p>
+  <form action="/api/auth/signin" method="post">
+    <label for="email" for="email">Email</label>
+    <input type="email" name="email" id="email" />
+    <label for="password">Password</label>
+    <input type="password" name="password" id="password" />
+    <button type="submit">Login</button>
+  </form>
+</Layout>
+```
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/backend/supabase.mdx:294]:
```suggestion
<Layout title="Connexion">
  <h1>Connexion</h1>
  <p>Nouveau par ici ? <a href="/register">Se créer un compte</a></p>
  <form action="/api/auth/signin" method="post">
    <label for="email" for="email">Email</label>
    <input type="email" name="email" id="email" />
    <label for="password">Mot de passe</label>
    <input type="password" name="password" id="password" />
    <button type="submit">Se connecter</button>
  </form>
</Layout>
```
```

File: src/content/docs/fr/guides/backend/supabase.mdx [hunk 3]
```diff
@@ -0,0 +1,502 @@
...
+    auth: {
+      flowType: "pkce",
+    },
+  },
+);
+```
+
+Ensuite, dans le tableau de bord de Supabase, activez le fournisseur OAuth que vous souhaitez utiliser. Vous pouvez trouver la liste des fournisseurs supportés dans l'onglet **Authentication > Providers** de votre projet Supabase.
+
+L'exemple suivant utilise GitHub comme fournisseur OAuth. Pour connecter votre projet à GitHub, suivez les étapes de la [documentation Supabase] (https://supabase.com/docs/guides/auth/social-login/auth-github).
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/backend/supabase.mdx:359]:
```suggestion
L'exemple suivant utilise GitHub comme fournisseur OAuth. Pour connecter votre projet à GitHub, suivez les étapes de la [documentation Supabase](https://supabase.com/docs/guides/auth/social-login/auth-github).
```

File: src/content/docs/fr/guides/backend/supabase.mdx [hunk 4]
```diff
@@ -0,0 +1,502 @@
...
+  <p>New here? <a href="/register">Créer un compte</a></p>
+  <form action="/api/auth/signin" method="post">
+    <label for="email" for="email">Email</label>
+    <input type="email" name="email" id="email" />
+    <label for="password">Password</label>
+    <input type="password" name="password" id="password" />
+    <button type="submit">Login</button>
+    <button value="github" name="provider" type="submit">Se connecter avec GitHub</button>
+  </form>
+</Layout>
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/backend/supabase.mdx:420]:
```suggestion
<Layout title="Connexion">
  <h1>Connexion</h1>
  <p>Nouveau par ici ? <a href="/register">Créer un compte</a></p>
  <form action="/api/auth/signin" method="post">
    <label for="email" for="email">Email</label>
    <input type="email" name="email" id="email" />
    <label for="password">Mot de passe</label>
    <input type="password" name="password" id="password" />
    <button type="submit">Login</button>
    <button value="github" name="provider" type="submit">Se connecter avec GitHub</button>
  </form>
</Layout>
```

File: src/content/docs/fr/guides/backend/supabase.mdx [hunk 5]
```diff
@@ -0,0 +1,502 @@
...
+
+    if (error) {
+      return new Response(error.message, { status: 500 });
+    }
+
+    return redirect(data.url);
+  }
+
+  if (!email || !password) {
+    return new Response("Email and password are required", { status: 400 });
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/backend/supabase.mdx:454]:
```suggestion
    return new Response("L'adresse e-mail et le mot de passe sont nécessaires", { status: 400 });
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Some translations in examples and one link breaked

**Comment by McFlyPartages** [comment]
> Some translations in examples and one link breaked

Merci c'est corrigé

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Review by dreyfus92 — APPROVED** [review]
Merci à tous, LGTM ✅

**Pull request merged**